spec/lib/api/v3/utilities/path_helper_spec.rb

```diff
@@ -54,7 +54,21 @@
 
     it_behaves_like 'api v3 path'
 
-    it { is_expected.to match(/^\/api\/v3\/attachments\/1/) }
+    it { is_expected.to eql('/api/v3/attachments/1') }
+  end
+
+  describe '#attachment_download' do
+    subject { helper.attachment_download 1 }
+
+    it { is_expected.to eql('/attachments/1') }
+  end
+
+  describe '#attachments_by_work_package' do
+    subject { helper.attachments_by_work_package 1 }
+
+    it_behaves_like 'api v3 path'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.